### ISSUE TITLE
Wix v4 Burn - Image Control in a page is visible in other pages

### ISSUE THREAD

**Issue by wmanning** [wixtoolset/issues#7374]
## Bug

* Which version of WiX are you building with?

4.0.0-rc.4

* Which version of Visual Studio are you building with (if any)?

2019 (16.11.25)

* Which version of the WiX Toolset Visual Studio Extension are you building with (if any)?

Heatwave 1.0.0

* Which version of .NET are you building with?

N/A

* If the problem occurs when installing your packages built with WiX, what is the version of Windows the package is running on?

Windows 10 Pro

* Describe the problem and the steps to reproduce it.

Have a custom StdWixBA theme.  If I add a ImageControl under the Install Page it is showing up in some of the other pages such as Modify and Modify progress. Does not show up under install progress or install success though.
See discussion here: https://github.com/orgs/wixtoolset/discussions/7373 

* Describe the behavior you expected and how it differed from the actual behavior.

ImageControl should under a particular page should not show up in other pages.

A work around is to add ```VisibleCondition="NOT WixBundleInstalled"``` to the ImageControl.

Note: will add link to repro in a week or two.

**Comment by wmanning** [comment]
Yep Removing Visible="yes" works. No longer need the VisibleCondition="NOT WixBundleInstalled" condition.

**Comment by barnson** [comment]
Thanks. Will need to rethink `Visible` for paged controls.
